src/nl/hannahsten/texifyidea/util/Magic.kt

```diff
@@ -229,15 +229,42 @@ object Magic {
         /**
          * Environments that introduce figures
          */
+        @JvmField
         val figures = hashSetOf("figure")
 
         @JvmField
         val verbatim = hashSetOf("verbatim", "Verbatim", "lstlisting", "plantuml", "luacode", "luacode*")
 
+        @JvmField
         val algorithmEnvironments = setOf("algorithmic")
+
+        /**
+         * All environments that define a matrix.
+         */
+        @JvmField
+        val matrixEnvironments = setOf(
```

Comment:
- Missing the starred versions of `[p|b|v|B|V|]matrix`, as provided by the `mathtools` package, see [Section 3.4.1 of the docs](http://texdoc.net/texmf-dist/doc/latex/mathtools/mathtools.pdf).
- Missing `[p|b|v|B|V]smallmatrix` and their starred versions (also `mathtools`)
- Maybe also add `gmatrix` (no stars) from the `gauss` package (wouldn't hurt this one, but there are probably more packages that provide fancy matrices that I am not aware of).

Reply:
Ah yes. The classic _hey this is also a thing somewhere and I dont know about it_ scenario. will add.